modules/Leads/views/view.convertlead.php

```diff
@@ -509,6 +510,23 @@ protected function handleSave()
             $lead->converted = '1';
             $lead->in_workflow = true;
             $lead->save();
+
+            // handle copying photo file, if present:
+            if (!empty($lead->id) && !empty($lead->photo) &&
+                !empty($beans['Contacts']->id) && !empty($beans['Contacts']->photo)) {
+                $bCopied = false;
+                if (($lead->photo == $beans['Contacts']->photo) && is_readable('upload/' . $lead->id . '_photo')) {
```

Comment:
Can ```===``` be used here?

Reply:
There isn't much question about the data types of these Bean fields, but ok, I'll change it to `===`

Reply:
Yeah it doesn't really matter here, it's simply to resolve the inspection from appearing.